sdk/main/include/Channel.h

```diff
@@ -69,6 +83,26 @@ class Channel
    */
   void initChannel(const std::string& url);
 
+  /**
+   * Submit a query request on this channel.
+   *
+   * @param query   The query to send.
+   * @param timeout The timeout duration.
+   * @return The protobuf response object and the gRPC status.
+   */
+  std::pair<proto::Response, grpc::Status> submitRequest(const proto::Query& request,
+                                                         const std::chrono::duration<double>& timeout);
+
+  /**
+   * Submit a transaction request on this channel.
+   *
+   * @param transcation The transaction to send.
+   * @param timeout     The timeout duration.
+   * @return The protobuf response object and the gRPC status.
+   */
+  std::pair<proto::TransactionResponse, grpc::Status> submitRequest(const proto::Transaction& request,
```

Comment:
This function has to remain overloaded so that the correct version can still be called in `Executable.cc`. See line 55 in that file. Agreed the parameter names can be more clear, I actually fix that in another PR.